include/tensor.cuh

```diff
@@ -1188,7 +1188,9 @@ Nullspace<T>::Nullspace(DTensor<T> &a) {
     for (size_t i = 0; i < nMats; i++) { // for each matrix
         // Slice the matrix of left SVs to get the matrix that spans
         // the nullspace of a[:, :, i]
-        unsigned int nullityI = n - hostRankA[i]; // nullity(A[:, :, i])
+        unsigned int rankAi = hostRankA[i];
```

Comment:
why split into two lines ?

Reply:
I thought I needed to use `rankAi`, but then I realised I didn't. It's all the same for the compiler and I'm lazy to change back it now :)